examples/play.py

```diff
@@ -1,8 +1,18 @@
+from pathlib import Path
+from PIL import Image
 import argparse
 import pystk
 from time import time
+import numpy as np
```

Comment:
do you actually need numpy? Or can pil use the memoryview?

Reply:
PIL didn't work

"memoryview object has no attribute '__array_interface__'